docs/components/modules/ROOT/pages/bean-component.adoc

```diff
@@ -167,7 +167,7 @@ from("direct:start").bean(ExampleBean.class);
 How bean methods to be invoked are chosen (if they are not specified
 explicitly through the *method* parameter) and how parameter values are
 constructed from the Message are all defined by the
-Bean Binding mechanism which is used throughout
+https://camel.apache.org/manual/latest/bean-binding.html[Bean Binding] mechanism which is used throughout
```

Comment:
We're using [Antora](https://antora.org/) to build the documentation, so we need to adhere to the [xref syntax](https://docs.antora.org/antora/2.3/page/page-id/#whats-a-cross-reference) for linking to pages.

```suggestion
xref:ROOT:bean-binding.adoc[Bean Binding] mechanism which is used throughout
```

Reply:
@zregvart Thank you